java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -1677,4 +1681,53 @@ public CompletableFuture<Long> sintercard(@NonNull String[] keys, long limit) {
                         new String[] {SET_LIMIT_REDIS_API, Long.toString(limit)});
         return commandManager.submitNewCommand(SInterCard, arguments, this::handleLongResponse);
     }
+
+    @Override
+    public CompletableFuture<String[]> sort(@NonNull String key) {
+        return commandManager.submitNewCommand(
+                Sort,
+                new String[] {key},
+                response -> castArray(handleArrayResponse(response), String.class));
+    }
+
+    @Override
+    public CompletableFuture<String[]> sort(
+            @NonNull String key, @NonNull SortBaseOptions sortBaseOptions) {
```

Comment:
after renaming the options, you will have SortClusterOptions here... and you'll see that it makes more sense to move this signature to the RedisClusterClient.  
Cluster client (RedisClusterClient) gets cluster options (SortClusterOptions). 
Standalone RedisClient gets standalone options (SortOptions)
and the BaseClient gets signatures without options.